dockerfiles/QAT/README.md

```diff
@@ -0,0 +1,59 @@
+
+  * [Introduction](#introduction)
+  * [Building](#building)
+  * [Options](#options)
+
+## Introduction
+
+The files in this directory can be used to build a modified Kata Containers rootfs
+and kernel with modifications to support Intel QAT hardware.
+
+The generated files will need to be copied and configured into your Kata Containers
+setup.
+
+Please see the
+[Kata QAT documentation](https://github.com/kata-containers/documentation/blob/master/use-cases/using-Intel-QAT-and-kata.md)
+for more specific QAT details.
+
+## Building
+
+The process is executed using Docker:
+
+```sh
+$ cd ..
+$ docker build --label kataqat --tag kataqat:latest QAT
+$ mkdir ./output
+$ docker run -ti --rm --privileged -v /dev:/dev -v $(pwd)/output:/output kataqat
+```
+
+> **Note:** The use of the `--privileged` and `-v /dev:/dev` arguments to the `docker run` are
+> necessary, to enable the scripts within the container to generate a roofs file system.
+
+When complete, the generated files will be placed into the output directory, for example:
+
+```sh
+$ ls -l output
+total 136236
+-rw-r--r-- 1 root root 134217728 Apr  1 16:26 kata-containers.img
+-rw-r--r-- 1 root root   5284400 Apr  1 16:26 vmlinuz-kata-linux-5.4.15-69_qat
+```
+
+## Options
+
+A number of parameters to the scripts are configured in the `Dockerfile`, and thus can be modified
+on the commandline.
+
+
+| Variable | Definition | Default value |
+| -------- | ---------- | ------------- |
+| QAT_DRIVER_VER | QAT driver version to use | qat1.7.l.4.9.0-00008.tar.gz |
+| QAT_DRIVER_URL | URL to curl QAT driver from | https://01.org/sites/default/files/downloads/${QAT_DRIVER_VER} |
+| QAT_COMPILE_OPTIONS | `configure` options for QAT driver | --disable-qat-lkcf --enable-icp-sriov=guest |
+| KATA_VERSION | Kata Branch or Tag to build from | master |
+| OUTPUT_DIR | Directory inside container where results are stored | /output |
+
+Variables can be set on the `docker run` commandline, for example:
```

Comment:
podman is also supported, isn't it?

Reply:
No idea. I'm not set up to run it. It is a **Dockerfile** ;-) - if somebody wants to try/confirm that this also works with other container builder/managers, I or they can add it to the doc...